### DISCUSSION TITLE
WiX v5: -help doesn't work on WiXStandardBA

### DISCUSSION THREAD

**Discussion by ADD-Izan-Garcia** [wixtoolset/issues#8564] [Questions]
WiX version:
`5.0.0`
.NET version:
`.NET 8.0.300`
HeatWave version:
`1.0.4.5`
Windows Version:
`Windows Server 2022 21H2`

**Problem:**
We had before WiX v3 and it worked, but when migrating to WiX v5 the -help option doesn't work.

**Repro steps:**
1. Have a Bundle.wxs
2. Have the bal extension on your setup.bootstrapper.wixproj
3. Use the theme hyperLinkLargeLicense
4. Have a chain with the NetFx472Redist, the NetHostingRedist exe and an .msi
5. Execute the BA .exe with the -help command

**Actual Result:**
An empty dialog with only the name of the BA App.

**Expected Result:**
Help dialog with the HelpText: "/install | /repair | /uninstall ..."

**Comment by kumarashokd** [comment]
@ADD-Izan-Garcia  I also face same issue , details https://github.com/orgs/wixtoolset/discussions/8604

**Comment by cz-dev-ge** [comment]
see https://github.com/wixtoolset/issues/issues/8808
